src/shared/services/path-util.service.ts

```diff
@@ -26,6 +26,11 @@ import { Injectable } from '@angular/core';
 @Injectable()
 export class PathUtilService {
 
+  // to make sure it's constant/final
```

Comment:
I see, was missing the Java terminology. Yes, I think we can just remove the comment.